addons/website_event_track/controllers/event_track.py

```diff
@@ -122,7 +121,7 @@ def _event_tracks_get_values(self, event, tag=None, **searches):
             ])
 
         # fetch data to display with TZ set for both event and tracks
-        now_tz = utc.localize(fields.Datetime.now().replace(microsecond=0), is_dst=False).astimezone(timezone(event.date_tz))
+        now_tz = fields.Datetime.now().replace(microsecond=0, tzinfo=UTC).astimezone(ZoneInfo(event.date_tz))
```

Comment:
Nitpick. fields.Datetime.now already strips microseconds and is naive.
```suggestion
        now_tz = fields.Datetime.now().astimezone(ZoneInfo(event.date_tz))
```

Reply:
>  fields.Datetime.now [...] is naive.

which is exactly why it should be localised first.

Reply:
We run in UTC, so it does not have to be. But you can keep it as is.